sys-kernel/coreos-firmware/coreos-firmware-99999999.ebuild

```diff
@@ -166,9 +166,10 @@ src_prepare() {
 }
 
 src_install() {
-	if use !savedconfig; then
-		save_config ${PN}.conf
-	fi
+	# Flatcar: Don't save the firmware config to /etc/portage/savedconfig/
```

Comment:
Why do we need this divergence in the ebuild file? Shouldn't the USE flag be enough?

Reply:
The use flag determines whether a saved config is used for the build. However, saving is done in the other case to have a base config to start editing from.

Reply:
Corrected the other comment and updated the commit message

Reply:
Thanks, I understand now.